docker-compose-native/docker-compose.yml

```diff
@@ -90,11 +101,29 @@ services:
     networks:
       presto:
         ipv4_address: 172.19.0.11
-networks:
-  presto:
-    driver: bridge
-    ipam:
-      driver: default
-      config:
-      - subnet:  172.19.0.0/24
-        gateway: 172.19.0.1
+
+  worker_2:
+    container_name: worker_2
+    image: public.ecr.aws/oss-presto/presto-native:0.289-ubuntu-arm64
+    platform: linux/arm64
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 2G
```

Comment:
this is 4G in a java cluster, can you match that?

Reply:
Thanks, updated.